### PR TITLE
Create maxplayers.js

### PR DESCRIPTION
**Pull request by andriycraft**

### CODE REVIEW

**Comment by andriycraft** [comment]
Done

**Comment by rom1504** [comment]
please add doc for this new option

**Comment by andriycraft** [comment]
@rom1504

File: src/lib/plugins/maxplayers.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+let playersonline = 0
```
Comment (rom1504) [inline src/lib/plugins/maxplayers.js:1]: Can you use a variable of the server instead of a global ?

**Comment by andriycraft** [comment]
@rom1504

File: src/lib/plugins/maxplayers.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+module.exports.player = async function (player, serv, settings) {
+  serv.playersonline = 0
```
Comment (rom1504) [inline src/lib/plugins/maxplayers.js:2]: that will reset the number of players whenever a new player joins

**Comment by rom1504** [comment]
you could use the existing players object to check the number of players

**Pull request closed without merging**